expression/function_traits.go

```diff
@@ -52,6 +41,23 @@ var unFoldableFunctions = map[string]struct{}{
 	ast.GetParam:  {},
 }
 
+// DeferredFunctions stores non-deterministic functions which can be deferred.
```

Comment:
How adding another comment to specify that the deferred functions take effects only when plan cache is enabled?

Reply:
Thank you for the comment. I've added the comment to the updated the PR.